packages/rpm/polymc.spec

```diff
@@ -0,0 +1,55 @@
+%global _origdir %(pwd)
+
+Name: polymc
+Version: 
+Release: 1%{?dist}
+Summary: A custom launcher for Minecraft
+License: GPLv3
+URL: https://polymc.org/
+
+BuildArch: x86_64
+BuildRequires:  java-devel
+BuildRequires:  pkgconfig(gl)
+BuildRequires:  pkgconfig(Qt5)
+BuildRequires:  pkgconfig(zlib)
+
+Requires: java-headless
+Requires: pkgconfig(gl)
+Requires: pkgconfig(Qt5)
+Requires: pkgconfig(zlib)
+
+%description
+A custom launcher for Minecraft that allows you to easily manage multiple installations of Minecraft at once
+
+%prep
+mkdir -p %{_builddir}/%{name}
+cp -r %{_origdir}/../../* %{_builddir}/%{name}
+
+%build
+cd %{_builddir}/%{name}
+cmake \
+  -DLauncher_LAYOUT=lin-system \
+  -DCMAKE_INSTALL_PREFIX=/usr \
+  -DLauncher_LIBRARY_DEST_DIR=%{_lib} \
+  .
+
+%cmake_build
```

Comment:
make sure it includes -j$(nproc)

Reply:
`%cmake_build` is a macro. it already contains another macro called `%{_smp_mflags}`, which automatically adds `-j$(nproc)` to the build flags, as well as others provided by the distribution.